ledger/acctupdates_test.go

```diff
@@ -172,6 +172,14 @@ func (ml *mockLedgerForTracker) BlockHdr(rnd basics.Round) (bookkeeping.BlockHea
 	return ml.blocks[int(rnd)].block.BlockHeader, nil
 }
 
+func (ml *mockLedgerForTracker) BlockTimeStamp(rnd basics.Round) int64 {
+	if rnd > ml.Latest() {
```

Comment:
Is Latest() the block being worked on, or the last block committed?  If being worked on, then we want >=.  (I have no idea, just double checking.)

Reply:
Latest() comes from ledger so it is the last committed.